chainercv/extensions/detection/detection_vis_report.py

```diff
@@ -210,62 +78,30 @@ def __call__(self, trainer):
         if not _available:
             return
 
-        for idx in self.indices:
-            formated_filename_base = osp.join(trainer.out, self.filename_base)
-            out_file = (formated_filename_base +
-                        '_idx={}'.format(idx) +
-                        '_iter={}'.format(trainer.updater.iteration) + '.jpg')
-
-            inputs = self.dataset[idx]
-            self._check_type_dataset(inputs)
+        label_names = getattr(self.dataset, 'labels', None)
```

Comment:
`label_names` should be passed as an argument.
The dataset loses attributes once wrapped by classes like TransformDataset.